components/actionbar/index.css

```diff
@@ -11,50 +11,10 @@
  * governing permissions and limitations under the License.
  */
 
-.spectrum-ActionBar {
-	/* Layout */
-	--spectrum-actionbar-height: var(--spectrum-action-bar-height);
-	--spectrum-actionbar-minimum-width: var(--spectrum-action-bar-minimum-width);
-	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-medium-size-extra-large);
-	--spectrum-actionbar-spacing-label-to-action-group: var(--spectrum-action-bar-label-to-action-group-area);
-	--spectrum-actionbar-spacing-padding-inline: var(--spectrum-action-bar-edge-to-content-area);
-	--spectrum-actionbar-spacing-top-area: var(--spectrum-action-bar-top-to-content-area);
-	--spectrum-actionbar-spacing-bottom-area: var(--spectrum-action-bar-bottom-to-content-area);
-	--spectrum-actionbar-close-button-to-counter: var(--spectrum-action-bar-close-button-to-counter);
-	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- defines the field label mod  */
-	--spectrum-actionbar-item-counter-line-height: var(--spectrum-line-height-100);
-	--spectrum-actionbar-item-counter-color: var(--spectrum-neutral-content-color-default);
-
-	/* colors - applied to popover */
-	--spectrum-actionbar-popover-background-color: var(--spectrum-background-elevated-color);
-	--spectrum-actionbar-popover-border-color: var(--spectrum-action-bar-border-color);
-
-	/* spacing of action bar bottom and horizontal outer edge */
-	--spectrum-actionbar-spacing-outer-edge: var(--spectrum-spacing-300);
-
-	/* drop shadow */
-	--spectrum-actionbar-shadow-horizontal: var(--spectrum-drop-shadow-elevated-x);
-	--spectrum-actionbar-shadow-vertical: var(--spectrum-drop-shadow-elevated-y);
-	--spectrum-actionbar-shadow-blur: var(--spectrum-drop-shadow-elevated-blur);
-	--spectrum-actionbar-shadow-color: var(--spectrum-drop-shadow-elevated-color);
-
-	&.spectrum-ActionBar--emphasized {
-		--spectrum-actionbar-item-counter-color: var(--spectrum-gray-25);
-		--spectrum-actionbar-popover-background-color: var(--spectrum-neutral-content-color-default);
-	}
-
-	/* cjk language support for item counter */
-	&:lang(ja),
-	&:lang(zh),
-	&:lang(ko) {
-		--spectrum-actionbar-item-counter-line-height-cjk: var(--spectrum-cjk-line-height-100);
```

Comment:
The line-height was only be used on the form-field so I moved that selector to below. It matches what form field does naturally though so maybe that's something we don't need to be overriding?